pybrake/test_queries.py

```diff
@@ -0,0 +1,75 @@
+import time
+
+from .queries import QueryStat, QueryStats
+
+
+def test_query_metric_start_end():
+    stats = QueryStat(time=1648551580.0367732)
+    assert stats.__dict__ == {
+        'count': 0,
+        'sum': 0,
+        'sumsq': 0,
+        'tdigest': 'AAAAAkAkAAAAAAAAAAAAAA==',
+        'query': '',
+        'method': '',
+        'route': '',
+        'time': '2022-03-29T10:59:00Z'
+    }
+
+
+def test_query_stats_performance_stats():
+    stats = QueryStats(**{"config": {
+        "performance_stats": False,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
+    }})
+    assert stats.notify(
+        query="SELECT * FROM foos",
+        method="GET",
+        route='test',
+        start_time=time.time(),
+        end_time=time.time(),
+    ) is None
+
+
+def test_query_stats_query_stats():
+    stats = QueryStats(**{"config": {
+        "performance_stats": True,
+        "query_stats": False,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
+    }})
+    assert stats.notify(
+        query="SELECT * FROM foos",
+        method="GET",
+        route='test',
+        start_time=time.time(),
+        end_time=time.time(),
+    ) is None
+
+
+def _test_query_stats_notify():
+    stats = QueryStats(**{"config": {
+        "performance_stats": True,
+        "query_stats": True,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
+    }})
+    stats.notify(
```

Comment:
If we test for `None` in the previous tests, we should probably be negating the assertion (not none, or the expected return value).

Reply:
When the `notify` method is correctly executed, it does not return anything.
As a result, I intend to rework the code in pybrake refresh. As a result, this type of circumstance will never occur.

Reply:
From what I understand, in Python, if you don't explicitly return a value, the function would return `None`. So when it is said "it doesn't return anything", it really returns `None`. Does that make sense?

Reply:
A test should assert something, right? How will this test ever fail if it doesn't assert anything?